mmdet/models/transformer_head/__init__.py

```diff
@@ -0,0 +1,8 @@
+from .transformer import (FFN, MultiheadAttention, Transformer,
```

Comment:
No need to create a dir for transformer. Simply move transformer.py into mmdet/models/utils/